Dockerfile

```diff
@@ -17,6 +17,7 @@ ENV NPM_RUN=start \
     NPM_CONFIG_LOGLEVEL=info \
     NPM_CONFIG_PREFIX=$HOME/.npm-global \
     PATH=$HOME/node_modules/.bin/:$HOME/.npm-global/bin/:$PATH \
+    npm_config_tarball=/usr/share/node/node-v${NODE_VERSION}-headers.tar.gz \
```

Comment:
Minor nit - very minor! But it seems odd to me to have a variable name in all lowercase when all of the others are in all caps. The inconsistency is a little odd to me.

Reply:
Not sure what I was thinking there, let me fix that. Thx